python/command_runner/interact.py

```diff
@@ -73,12 +73,14 @@ def ask_yna_question(self, message, force_prompt=False):
             # always clicked
             return True
 
-        res = QtGui.QMessageBox.question(
-            self._parent,
-            "Shotgun",
-            message,
-            QtGui.QMessageBox.Yes | QtGui.QMessageBox.No | QtGui.QMessageBox.YesToAll,
-        )
+        msgbx = QtGui.QMessageBox(self._parent)
+        msgbx.setWindowTitle("Shotgun")
+        msgbx.setIcon(msgbx.Icon.Question)
+        msgbx.setText(message)
+        msgbx.addButton(QtGui.QMessageBox.Yes)
+        msgbx.addButton(QtGui.QMessageBox.YesToAll)
+        msgbx.addButton(QtGui.QMessageBox.No)
+        res = msgbx.exec_()
```

Comment:
This was done, as I was seeing duplicate YesToAll buttons otherwise. I'm not sure why that was though.

Reply:
This has been fixed in core.